comments/admin.py

```diff
@@ -32,3 +32,7 @@ class CommentAdmin(CustomTranslationAdmin):
         "included_forecast",
         "is_private",
     ]
+
+    def should_update_translations(self, obj):
+        is_private = obj.on_post.default_project.default_permission is None
+        return not is_private
```

Comment:
for futureproofing in case permissions system changes, I'd suggest using the `filter_private` method on the PostQuerySet.

something like:
`is_private = Post.objects.filter_private().filter(id=obj.on_post_id).exists()`

Reply:
Ooh, I actually created a helper function on Post, but forgot to use it here. Will change it.

Reply:
looks live you've added an `is_private` method to the Post model. user that instead:
`return not obj.on_post.is_private()`

Reply:
haha simultaneous commenting. whoops, I never refreshed before the followup

Reply:
We also have `Post.objects.filter_private()` QuerySet method which could be reused